specs/http-client/index.html

```diff
@@ -429,117 +428,116 @@ <h3>Serializing the request content</h3>
+          binary item). If it is not, the error <a>err:HC003</a> MUST be raised.</p>
       </section>
     </section>
 
     <section>
-      <h2>Content types handling</h2>
-      <p>In both requests and responses, MIME type strings are used to choose the way the entity
-        content has to be respectively serialized or parsed.  Four different kinds of type are
-        defined here, which are used in the above text about sending request and receiving response.
-        The intent is to provide the spirit of the entity content handling regarding its content
-        type, but an implementation is encouraged to deviate from those rules if it is obvious that
-        a particular type should be treated in a specific way (normally, that would be the case only
-        to treat a binary type as another type).</p>
+      <h2>Processing Media Types</h2>
+      <p>In both requests and responses, Media Type strings are used to choose the way the entity
+        content has to be serialized or parsed.</p>
+      <p>We define four different classes of Media Type, which are used for sending requests and receiving responses.
+        The intent is to provide guidance as to handling the entity content with respect to its content
+        type, but an implementation is permitted to deviate from those rules if it is obvious that
+        a particular type should be treated in a specific way, typically this can be useful for binary types such as [[EXI]].</p>
       <ul>
-        <li>An XML media type has a MIME type of <code>text/xml</code>,
+        <li>For XML the following media types are appropriate: <code>text/xml</code>,
           <code>application/xml</code>, <code>text/xml-external-parsed-entity</code>,
           or <code>application/xml-external-parsed-entity</code>, as defined in [[!rfc3023]] (except
-          that <code>application/xml-dtd</code> is considered a text media type).  MIME types ending
-          by <code>+xml</code> are also XML media types.</li>
-        <li>An HTML media type has a MIME type of <code>text/html</code>.</li>
-        <li>Text media types are the remaining types beginning with <code>text/</code>.</li>
-        <li>Binary types are all the other types.  An implementation can treat some of those binary
+          that <code>application/xml-dtd</code> is considered a text media type). Media types ending
+          with <code>+xml</code> are also considered XML types.</li>
+        <li>For HTML the media type <code>text/html</code> is suggested.</li>
+        <li>For Text, a media type beginning with <code>text/</code> is suggested.</li>
+        <li>All other media types are treated as binary. An implementation MAY treat some of those binary
           types as either an XML, HTML or text media type if it is more appropriate (this is
           implementation-defined).</li>
       </ul>
     </section>
 
     <section class="appendix">
-      <h2>Summary of Error Conditions</h2>
+      <h2>Error Codes</h2>
```

Comment:
I kept the wording of the QT specs here.

Reply:
I don't agree on changing this back. The document itself mentions "error code" in many places, it never mentions "error condition". So to be consistent, we should use "Error Code".